agent/04_agent_configure.sh

```diff
@@ -17,9 +17,16 @@ early_deploy_validation
 
 export CLUSTER_NAMESPACE=${CLUSTER_NAMESPACE:-"cluster0"}
 
+function add_host_entry {
+    ip=${1}
+    hostname=${2}
+
+    echo "${hostname}">>"${OCP_DIR}"/hostname
+    echo "${ip} ${hostname}" | sudo tee -a /etc/hosts
```

Comment:
It looks like this is going to appending to /etc/hosts each time its called.  The entry should be removed after running agent-gather.sh